packages/docker-node/src/ckb-docker-network.ts

```diff
@@ -1,13 +1,18 @@
 import Docker, { ContainerCreateOptions } from 'dockerode'
-import { DockerNodeStartOptions } from './types'
+import { DockerNodeStartOptions, InfraScript } from './types'
 import { join } from 'node:path'
 import fs from 'node:fs'
+import path from 'node:path'
+import { Address, Indexer, RPC, commons, helpers, hd, Transaction, config } from '@ckb-lumos/lumos'
 
 const CKB_NODE_IMAGE = 'kuai/ckb-dev'
 const DOCKER_SOCKET_PATH = process.env.DOCKER_SOCKET || '/var/run/docker.sock'
 
 export class CkbDockerNetwork {
   private _docker: Docker
+  private _lumosConfig: config.Config = config.getConfig()
+  private _port = '8114'
+  private _host = 'localhost'
```

Comment:
They can be defined as `private field`

Ref: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Classes/Private_class_fields

Reply:
> They can be defined as `private field`
> 
> Ref: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Classes/Private_class_fields

I just want to keep them the same as what other private property defined before, like `_docker`. Or I will change the `_docker` to `#docker` in this PR?

Reply:
`#docker` would be better because it's native JavaScript while `private` is `TypeScript` and not forcely restrcited.